controller/controller.go

```diff
@@ -254,14 +254,16 @@ func (c *Controller) ProcessEvents(ctx context.Context, deleteChan chan watch.Ev
 
 				accessor, err := meta.Accessor(e.Object)
 				if err != nil {
-					c.logger.LogCtx(ctx, "level", "error", "message", fmt.Sprintf("cannot create accessor %T", e.Object), "stack", fmt.Sprintf("%#v", err))
+					c.logger.LogCtx(ctx, "level", "error", "message", fmt.Sprintf("failed to create accessor %T", e.Object), "stack", fmt.Sprintf("%#v", err))
 				} else {
 					ctx = setLoggerCtxValue(ctx, loggerKeyObject, accessor.GetSelfLink())
 					ctx = setLoggerCtxValue(ctx, loggerKeyVersion, accessor.GetResourceVersion())
 				}
 			}
 
+			c.logger.LogCtx(ctx, "level", "debug", "message", "reconciling object")
```

Comment:
Is the information already present in the `ctx` or would it make sense to add the name / identifier of the object that is being reconciled? Also the same thing below for similar lines.

Reply:
Batteries included.

```
D 01/24 14:11:55 processing object events | operatorkit/controller/controller.go:411 | controller=test-controller
D 01/24 14:11:55 integration-controlflow-test/nodeconfigs/test-obj reconciling object | operatorkit/controller/controller.go:287 | controller=test-controller | event=update | loop=0 | version=364
D 01/24 14:11:55 integration-controlflow-test/nodeconfigs/test-obj stop reconciliation due to finalizer added | operatorkit/controller/controller.go:351 | controller=test-controller | event=update | loop=0 | version=364
D 01/24 14:11:55 integration-controlflow-test/nodeconfigs/test-obj reconciled object | operatorkit/controller/controller.go:289 | controller=test-controller | event=update | loop=0 | version=364
```